src/js/components/help/helpContent.jsx

```diff
@@ -16,31 +16,40 @@ export default class HelpContent extends React.Component {
         super(props);
 
         this.rssPromise = null;
+        this.urlPromise = null;
 
         this.state = {
-            rssUrl: ''
+            rssUrl: '',
+            validationRulesUrl: '#',
+            domainValuesUrl: '#'
         };
     }
 
     componentDidMount() {
-        this.rssPromise = generateRSSUrl();
-        this.rssPromise.promise
-            .then((url) => {
+
+        // also load the remaining URLs
+        this.urlPromise = generateProtectedUrls();
+        this.urlPromise.promise
+            .then((urls) => {
                 this.setState({
-                    rssUrl: url
+                    rssUrl: urls['RSS_v1.0.xlsx'],
```

Comment:
So the endpoint is returning the keys in the response as the filename of whatever is in the S3 bucket for the static files. If it doesn't match to what's here, it'll think it wasn't part of the response. I think we need a better way to handle it.